samples/dashboards/basic/your-first-dashboard/demo.css

```diff
@@ -4,3 +4,9 @@
 #dashboard-col-0 {
     text-align: center;
 }
+
+@media screen and (max-width: 680px) {
```

Comment:
I prefer to use the `576px` that is standardized.

Reply:
Agree.